test/merchant_test.rb

```diff
@@ -19,4 +29,22 @@ def test_merchant_has_a_name
 
     assert_equal "Ace", merchant.name
   end
+
+  def test_find_items_returns_items_associated_with_merchant
+    result = @merchant_repository.merchants.first.items
+    assert_equal 10, result.count
+
+  end
+
+  def test_find_invoices_returns_invoices_associated_with_merchant
+    result = @merchant_repository.merchants[9].invoices
+    assert_equal 1, result.count
+  end
+
+  private
+
+  def fixtures_directory
+    File.expand_path('../../data/fixtures', __FILE__)
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.